src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -201,15 +201,15 @@ def get_stable_build_info():
   return stable_build_info
 
 
-def get_latest_artifact_info(branch, target, signed=False):
+def get_latest_artifact_info(branch, target, signed=False, stable_build=False):
   """Return latest artifact for a branch and target."""
   client = get_client()
   if not client:
     return None
 
   # TODO(https://github.com/google/clusterfuzz/issues/3950)
   # After stabilizing the Cuttlefish image, revert this
-  if environment.is_android_cuttlefish():
+  if environment.is_android_cuttlefish() and stable_build:
```

Comment:
Is the TODO still relevant?

Reply:
Thanks for clarifying Aditya. Can you please update https://github.com/google/clusterfuzz/issues/3950 accordingly?

Just tag this CL as a reference to "connect" this CL to that open issue.